circuits/cpp/barretenberg/cpp/src/barretenberg/polynomials/pow.test.cpp

```diff
@@ -12,15 +11,21 @@ TEST(SumcheckPow, FullPowConsistency)
     constexpr size_t d = 5;
 
     FF zeta = FF::random_element();
-    PowUnivariate<FF> pow_univariate(zeta);
 
+    PowUnivariate<FF> pow_univariate(zeta);
     std::array<FF, d> variables{};
     for (auto& u_i : variables) {
         u_i = FF::random_element();
         pow_univariate.partially_evaluate(u_i);
     }
-
-    FF expected_eval = proof_system::honk::power_polynomial::evaluate<FF>(zeta, variables);
+    
```

Comment:
Legible, by-hand test is better AND it decouples this test from a very large module.